batches.go

```diff
@@ -0,0 +1,111 @@
+package packngo
+
+import (
+	"fmt"
+)
+
+const batchBasePath = "/batches"
+
+// BatchService interface defines available batch methods
+type BatchService interface {
+	Get(batchID string, listOpt *ListOptions) (*Batch, *Response, error)
+	List(ProjectID string, listOpt *ListOptions) ([]Batch, *Response, error)
+	Create(projectID string, batches *InstanceBatchCreateRequest) (*Batch, *Response, error)
+}
+
+// Batch type
+type Batch struct {
+	ID                     string     `json:"id,omitempty"`
+	State                  string     `json:"state,omitempty"`
+	Quantity               int32      `json:"quantity,omitempty"`
+	CreatedAt              *Timestamp `json:"created_at,omitempty"`
+	Href                   string     `json:"href,omitempty"`
+	Project                Href       `json:"project,omitempty"`
+	Instances              Href       `json:"instances,omitempty"`
+	Facilities             []Facility `json:"facilities,omitempty"`
+	FacilityDiversityLevel int32      `json:"facility_diversity_level,omitempty"`
+}
+
+//BatchesList represents collection of batches
+type batchesList struct {
+	Batches []Batch `json:"batches,omitempty"`
+}
```

Comment:
omitempty makes no sense here I guess?

Reply:
What if there are no batches returned from the API?

Reply:
My bad, if you use the struct only for unmarshaling, then `omitempty` doesn't matter.